content/2-2.md

```diff
@@ -159,33 +160,34 @@ Optional ideas to support assessment.
 
 {% include excol.html type="middle" %}
 
-Explain use of HTML elements and WAI-ARIA attributes to indicate the presence of submenu items, for example, `aria-haspopup`. Explain how to convey visible or hidden states of submenu items using WAI ARIA or text in their labels to make them operable by all users.
+Explain the use of HTML elements and WAI-ARIA attributes to indicate the presence of submenu items, for example, `aria-haspopup`. Explain how to convey visible or hidden states of submenu items using WAI-ARIA to make them operable by all users.
```

Comment:
[Minor] Is this about communicating the state to users rather than making it operable?

Reply:
I would say it is about both. You definitely need to make these operable. But for some users, not communicating their state means they cannot operate them efficiently as they may not be even aware about the submenus and about their state. Any suggestions about how to make this clearer?

Reply:
I guess I fixated on the word 'operable' as one of the principles. If you used 'usable' it takes away from that issue and covers a lot of aspects. So:

> Explain how to convey visible or hidden states of submenu items using WAI-ARIA to make them usable by all users.